src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/eltwise.cpp

```diff
@@ -0,0 +1,442 @@
+                {{2, 1, 1, 3, 4}, {2, 17, 5, 3, 1}},
+        };
+        return inShapes_5D_Planar_Blocked;
+}
+
+const std::vector<CPUSpecificParams>& cpuParams_5D_Blocked_Planar() {
+        static const std::vector<CPUSpecificParams> cpuParams_5D_Blocked_Planar = {
+                CPUSpecificParams({nCdhw16c, ncdhw}, {nCdhw16c}, {}, {}),
+        };
+        return cpuParams_5D_Blocked_Planar;
+}
+
+const std::vector<CPUSpecificParams>& cpuParams_5D_Planar_Blocked() {
+        static const std::vector<CPUSpecificParams> cpuParams_5D_Planar_Blocked = {
+                CPUSpecificParams({ncdhw, nCdhw16c}, {nCdhw16c}, {}, {}),
+        };
+        return cpuParams_5D_Planar_Blocked;
+}
+
+const std::vector<CPUSpecificParams> & cpuParams_4D_1D_Constant_mode_x64() {
+        static const std::vector<CPUSpecificParams> cpuParams_4D_1D_Constant_mode = {
+                CPUSpecificParams({nChw16c, nchw}, {nChw16c}, {}, {})
+        };
+        return cpuParams_4D_1D_Constant_mode;
+}
+
+const std::vector<fusingSpecificParams> fusingParamsSet_x64{
+    // eltwise
+    fusingSigmoid,
+    fusingPRelu1D,
+    // depthwise
+    fusingReluScaleShift,
+    // fake quantize
+    fusingFakeQuantizePerTensorRelu,
+    fusingFakeQuantizePerChannelRelu,
+    fusingFQPerChannelSigmoidFQPerTensor
+};
+
+const auto params_4D = ::testing::Combine(
+        ::testing::Combine(
+                ::testing::ValuesIn(static_shapes_to_test_representation(inShapes_4D())),
+                ::testing::ValuesIn(eltwiseOpTypesBinInp()),
+                ::testing::ValuesIn(secondaryInputTypes()),
+                ::testing::ValuesIn(opTypes()),
+                ::testing::ValuesIn(netType()),
+                ::testing::Values(ov::element::undefined),
+                ::testing::Values(ov::element::undefined),
+                ::testing::Values(CommonTestUtils::DEVICE_CPU),
+                ::testing::Values(additional_config())),
+        ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D())),
+        ::testing::Values(emptyFusingSpec));
+
+INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_MemOrder_x64, EltwiseLayerCPUTest, params_4D, EltwiseLayerCPUTest::getTestCaseName);
+
+const auto params_4D_fusing = ::testing::Combine(
+        ::testing::Combine(
+                ::testing::ValuesIn(static_shapes_to_test_representation(inShapes_4D_fusing())),
+                ::testing::ValuesIn(eltwiseOpTypesBinInp()),
+                ::testing::Values(ngraph::helpers::InputLayerType::PARAMETER),
+                ::testing::ValuesIn(opTypes()),
+                ::testing::Values(ElementType::f32),
+                ::testing::Values(ov::element::undefined),
+                ::testing::Values(ov::element::undefined),
+                ::testing::Values(CommonTestUtils::DEVICE_CPU),
+                ::testing::Values(additional_config())),
+        ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D())),
+        ::testing::ValuesIn(fusingParamsSet_x64));
```

Comment:
Myabe I am missing smt, but I cannot find instances of blocked layout with non-empty fusing spec for x64. Previously they were covered due to the fact cpuParams_4D contained blocked layout inside